front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -116,6 +116,10 @@ const SimulationResults = ({
     }
   }, [extViewport]);
 
+  if (!selectedTrainSchedule && !projectionData) {
```

Comment:
Should we be more restrictive ? Sometimes there is a bug where we see the simulation results but not the space time chart (and we need to change the projection in order to display it)
```suggestion
  if (!selectedTrainSchedule || !projectionData) {
```

Reply:
Ah I had in mind that as the SpaceTimeChart does not depend on the selected train, it should not disappear if the simulation of the selectedTrain is loading.

But I can hide all the results below the SpaceTimeChart if we have no selected train ?

Reply:
Why not, but we shouldn't do that for "no projection" (see other comment)